### DISCUSSION TITLE
Configuring Formula column as primary value [Sep 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3751] [Discord Threads]
> 🙋‍♂️ **Nils**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported at the moment.  
#  
> 💁 **o1lab**  
#### Few ppl have requested this, we will think about it.  
#  
> 🙋‍♂️ **Nils**  
#### Would be cool to this see this in NocoDB 👍  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1016276229294407751)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by magori** [reply]
I have the same need to display a person (First name, Last name, Age).  
For now i use the Formula column  because it not possible for the Lookup column to have multiple child column.
It Will be alos nice if we can choose multiple child column for the Lookup column
